src/core/Expander/ExpanderGroup/ExpanderGroup.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { Component, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import { HtmlDiv, HtmlButton, HtmlButtonProps, HtmlSpan } from '../../../reset';
+import { baseStyles } from './ExpanderGroup.baseStyles';
+import { VisuallyHidden } from '../../../components';
+
+const baseClassName = 'fi-expander-group';
+const openClassName = `${baseClassName}--open`;
+const expandersContainerClassName = `${baseClassName}_expanders`;
+const openAllButtonClassName = `${baseClassName}_all-button`;
+interface InternalExpanderGroupProps {
+  /** Expanders and option other components */
+  children: ReactNode;
+  /** 'Open all' button text */
+  OpenAllText: string;
+  /** 'Close all'-component (Button) */
+  CloseAllText: string;
```

Comment:
Should the JSDoc have same logic?

> 'Open all' button text
> 'Close all' button text

Reply:
Fixed. Original wording was left there by mistake.